src/test/util/net.cpp

```diff
@@ -60,6 +60,7 @@ std::vector<NodeEvictionCandidate> GetRandomNodeEvictionCandidates(int n_candida
             /*m_network=*/ALL_NETWORKS[random_context.randrange(ALL_NETWORKS.size())],
             /*m_perm_flags=*/NetPermissionFlags::None,
             /*m_conn_type=*/ConnectionType::INBOUND,
+            /*m_blocks_in_flight=*/(int)random_context.randrange(5),
```

Comment:
I think `MAX_BLOCKS_IN_TRANSIT_PER_PEER` would be more reasonable here?

Reply:
Yes, i couldn't use the `MAX_BLOCKS_IN_TRANSIT_PER_PEER` constant here but i changed to the right value and left a comment.